### PR TITLE
delete resource  will return json fearture if result has body or data

### PR DESCRIPTION
**Pull request by slalx**
delete resource  will return json fearture if result has body or data

### CODE REVIEW

File: lib/api.js [hunk 1]
```diff
@@ -148,7 +148,7 @@ module.exports = (options, routeConfig) => {
       return;
     }
 
-    if (fname === 'destroy') {
+     if (fname === 'destroy' && !this.data && !this.body) {
```
Comment (fengmk2) [inline lib/api.js]: 多了一个空格

**Comment by dead-horse** [comment]
http://stackoverflow.com/questions/6581285/is-a-response-body-allowed-for-a-http-delete-request  按照规范是需要支持的，但是要发大版本

**Comment by fengmk2** [comment]
我手动合并。

**Comment by fengmk2** [comment]
land 9f8cedf620c261c550c8a92608ba8600a353bd3a

**Comment by fengmk2** [comment]
1.3.0

**Pull request closed without merging**